### PR TITLE
Add rawdata epa with facilities

### PR DESCRIPTION
**Pull request by dav1do**
Created a rawdata app to store data before it's formatted into whatever we need for the front end (plan to move to a new table in app likely). The schema currently doesn't enforce FKs on values and is just raw but it should be able to look up and join (or we can change it to be more explicit). Added a couple tasks that get data from the EPA web API ([facility](https://echo.epa.gov/tools/web-services/facility-search-all-data) for location info and [SDWA](https://echo.epa.gov/tools/web-services/facility-search-drinking-water) for violations etc. First we store the data in local csv files (allows everyone to use the data without having to ask for it) and there's a task to import it into the DB. Eventually, we'll want to schedule these somehow so they run every quarter or so and need to create something to update the final app table with the necessary location/violation data.

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 if __name__ == "__main__":
     dotenv.read_dotenv()
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "waterquality.settings")
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "settings.dev")
```
Comment (dav1do) [inline manage.py:9]: not sure about this but it wouldn't run when the settings file didn't exist

**Comment by dav1do** [comment]
Not sure if I should use the 'squash and merge' feature, squash now or wait until review. @mosesmc52 let me know what you prefer.

File: manage.py (hunk 1 above)
Comment (dav1do) [reply manage.py:9]: error was ```  File "/Users/david/.virtualenvs/venv/lib/python2.7/site-packages/django/conf/__init__.py", line 110, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named settings```

Comment (mosesmc52) [reply manage.py:9]: That makes sense. You inserted the correct settings path

**Comment by mosesmc52** [comment]
Squash and merge makes sense

**Pull request merged**